src/Raven.Client/Documents/Session/DocumentSession.Load.cs

```diff
@@ -25,7 +25,7 @@ public partial class DocumentSession
         /// <inheritdoc />
         public T Load<T>(string id)
         {
-            if (id == null)
+            if (string.IsNullOrWhiteSpace(id))
```

Comment:
I think you need to add this to `AsyncDocumentSession.LoadAsync` as well

Reply:
of course